updateStableIds/src/main/java/org/reactome/release/update_stable_ids/Main.java

```diff
@@ -0,0 +1,66 @@
+package org.reactome.release.update_stable_ids;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.persistence.TransactionsNotSupportedException;
+
+/**
+ * This function iterates through all instances, and checks if it has been changed since the previous release.
+ * Instances that have been changed have their 'identifierVersion' attribute incremented by 1 to reflect a change. 
+ *
+ */
+public class Main 
+{
+	private static final Logger logger = LogManager.getLogger();
+	
+    public static void main( String[] args ) throws Exception
+    {
+    	logger.info("Beginning UpdateStableIds step...");
+    	// This utility was meant to serve the same function as 'restore_database.pl'. 
+    	// For now, we don't want the Java to do the DB archiving, so this won't be used. 
+//       DatabaseUtils.archiveUsedDatabases("test_slice_67", "gk_central");
+       
+       String pathToConfig = "src/main/resources/config.properties";
```

Comment:
Users should be able to specify the path to the properties file from the command line. It's Ok to have a value like this as a _default_, but it should be possible for users to override this, if necessary.